src/commands.c

```diff
@@ -45,12 +42,10 @@ struct _UzblCommands {
     gboolean         search_forward;
     gchar           *search_text;
 
-#ifdef USE_WEBKIT2
 #if WEBKIT_CHECK_VERSION (2, 5, 1)
```

Comment:
And we definitely need 2.4 (for [this function](http://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebContext.html#webkit-web-context-set-web-extensions-initialization-user-data)).